test_scripts/API/AutoCompleteList/001_ATF_SetGlobalProperties_P_AutoCompleteList_OmitedText_HAPPY_PATH.lua

```diff
@@ -0,0 +1,75 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL to omit "autoCompleteText" param at request to HMI
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param and without (omited) <autoCompleteText> param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList> and <autoCompleteText>
+-- 3. SDL respond without <autoCompleteText>  with <resultCode_received_from _HMI> to mobile app
```

Comment:
@Aleksandriys, looks like you missed "Expected result" in header. 
I guess, at least p3."SDL respond without <autoCompleteText>  with <resultCode_received_from _HMI> to mobile app" Should be in expected result.